nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-client-dto/src/main/java/org/apache/nifi/web/api/entity/ClusterSummaryEntity.java

```diff
@@ -24,7 +24,7 @@
  * A serialized representation of this class can be placed in the entity body of a request or response to or from the API. This particular entity holds a reference to a ClusterSummaryDTO.
  */
 @XmlRootElement(name = "clusterSummaryEntity")
-public class ClusteSummaryEntity extends Entity {
+public class ClusterSummaryEntity extends Entity {
```

Comment:
I know this seems a bit nit-picky but I think we need to hold off renaming this class since we try to keep the nifi-client-dto backwards compatible between minor releases, and changing the name means any code written against this class would need to be updated, so I think we can hold off on that part and make that change during a major release like 2.0.0.

Reply:
I wasn't sure about that, so I created in a separate commit, reverting.